include/boost/beast/http/detail/rfc7230.ipp

```diff
@@ -80,15 +80,14 @@ char
 is_token_char(char c)
 {
     /*
-        tchar = "!" | "#" | "$" | "%" | "&" |
-                "'" | "*" | "+" | "-" | "." |
-                "^" | "_" | "`" | "|" | "~" |
+        tchar = "!" | "#" | "$" | "%" | "&" | "'" | "*" | "+" |
```

Comment:
This function is used by the HTTP parser. Changing it would change what is allowed in HTTP headers.